python_modules/dagster/dagster/generate/new_project/README.md

```diff
@@ -4,13 +4,13 @@ Welcome to your new Dagster repository.
 
 ### Contents
 
-| Name | Description |
-|-|-|
-| `{{ repo_name }}/` | A Python module that contains code for your Dagster repository |
-| `{{ repo_name }}_tests/` | A Python module that contains tests for `{{ repo_name }}` |
-| `workspace.yaml` | A file that specifies the location of the user code for Dagit and the Dagster CLI |
-| `README.md` | A description and guide for this code repository |
-| `setup.py` | A build script with Python package dependencies for this code repository |
+| Name                     | Description                                                                       |
+| ------------------------ | --------------------------------------------------------------------------------- |
+| `README.md`              | A description and guide for this code repository                                  |
+| `setup.py`               | A build script with Python package dependencies for this code repository          |
+| `workspace.yaml`         | A file that specifies the location of the user code for Dagit and the Dagster CLI |
+| `{{ repo_name }}/`       | A Python module that contains code for your Dagster repository                    |
```

Comment:
I know you didn't introduce this, but "Python module" is inaccurate. It it should say "Python page", "folder", or "directory" instead.